### PR TITLE
HooperFly airframe updates in support of motor mixing mods

### PR DESCRIPTION
**Pull request by hooperfly**
Updated the HooperFly specific airframes with motor mixing related modifications.

### CODE REVIEW

File: conf/airframes/HooperFly/racerpex_hexa_lisa_mx_20.xml [hunk 1]
```diff
@@ -47,6 +49,17 @@
     <load name="gps_ubx_ucenter.xml"/>
     <load name="geo_mag.xml"/>
     <load name="air_data.xml"/>
+    <load name="nav_survey_rectangle_rotorcraft.xml">
+      <define name="RECTANGLE_SURVEY_DEFAULT_SWEEP" value="10"/>
+    </load>
+    <load name="nav_survey_poly_rotorcraft.xml"/>
```
Comment (flixr) [inline conf/airframes/HooperFly/racerpex_hexa_lisa_mx_20.xml]: we don't have this module in the main repo yet....

**Comment by hooperfly** [comment]
@f

File: conf/airframes/HooperFly/racerpex_hexa_lisa_mx_20.xml (hunk 1 above)
Comment (hooperfly) [inline conf/airframes/HooperFly/racerpex_hexa_lisa_mx_20.xml]: You are refering to `nav_survey_poly_rotorcraft`, correct? I will comment that module out of each file. Thanks for catching that.

**Comment by flixr** [comment]
Thx, merged with 8e5fff89149056836a4786ffd9ae34115f34ce0a

**Pull request closed without merging**